### PR TITLE
Create priyankarpal.json

### PR DESCRIPTION
**Pull request by priyankarpal**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot from 2023-01-23 23-35-01
Screenshot from 2023-01-23 23-38-13

## Note to reviewers

I added my all links. Please check & if there is any problem then, please share. 

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3858"></a>

### CODE REVIEW

File: data/priyankarpal.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+       {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://linkedin.com/in/priyankarpal",
+      "icon": "FaLinkedin"
+    },
+       {
+      "name": "Instagram: Let's connect",
+      "url": "https://instagram.com/priyankarpal1",
+      "icon": "FaInstagram"
+    },
```
Comment (ChinmayMhatre) [inline data/priyankarpal.json:27]:
```suggestion
    }
```
There was an extra comma . You can commit this change

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
I have suggested some changes please take a look

**Review by ChinmayMhatre — APPROVED** [review]
Awesome! Looks good.

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**